app/packages/core/src/components/Grid/Grid.tsx

```diff
@@ -46,6 +47,15 @@ function Grid() {
   const setSample = fos.useExpandSample(store);
   const getFontSize = useFontSize(id);
 
+  const getCurrentActiveLabelFields = useRecoilCallback(
+    ({ snapshot }) =>
+      () => {
+        return snapshot
+          .getLoadable(fos.activeLabelFields({ modal: false }))
+          .getValue();
+      }
+  );
```

Comment:
Note, this callback is created on every render. Intentional?

Reply:
good catch. nope! will fix